basyx/aas/adapter/xml/xml_deserialization.py

```diff
@@ -388,15 +388,15 @@ def _child_text_mandatory_mapped(parent: etree.Element, child_tag: str, dct: Dic
     return _get_text_mandatory_mapped(_get_child_mandatory(parent, child_tag), dct)
 
 
-def _get_modeling_kind(element: etree.Element) -> model.ModelingKind:
+def _get_kind(element: etree.Element) -> model.ModellingKind:
     """
     Returns the modeling kind of an element with the default value INSTANCE, if none specified.
```

Comment:
here is another instance of "modeling"

Reply:
This here was done to get consistent with the JSON deserialization, where the same function is also called `get_kind`. Both ways would work, since the attribute is called `kind` and the class `ModellingKind`, however we should have the same naming scheme for both JSON and XML. I don't have a particular preference.

Reply:
Yeah, I meant the docstrings